package.json

```diff
@@ -48,15 +45,19 @@
     "optimizecss": "postcss dist/css/style.css -c postcss.config.js -o dist/css/style.css",
     "scss": "node-sass src/scss/style.scss -o dist/css",
     "gutenberg": "node-sass src/scss/gutenberg-blocks-style.scss -o dist/css",
-    "lint": "eslint src/js || true",
-    "lint-scss": "stylelint src/scss/**/*.scss --syntax scss --config .stylelintrc || true",
+    "lint": "npm run lint-js && npm run lint-scss",
+    "lint:js": "npm run lint-js",
+    "lint:scss": "npm run lint-scss",
```

Comment:
I didn't delete the `lint-*` variants, as I'm unsure if they are in documentation, but to be consistent I think `lint:*` is more consistent.